### PR TITLE
fix(challenges): challenge description is formatted and concised

### LINKED ISSUE #193: Provide better instructions for 'has' method in Create a Map Data Structure challenge
#### Is your feature request related to a problem? Please describe.

https://learn.freecodecamp.org/coding-interview-prep/data-structures/create-a-map-data-structure/.
Instructions for 'has' method in the challenge is pretty ambiguous unlike descriptions of other methods in the same challenge. It lacks proper explanation of what the method is expecting.
 
#### Describe the solution you'd like

Change the text from "has returns a boolean for the presence or absence of an item" to "has accepts a key and returns true or false based on the presence of a key" 

#### Describe alternatives you've considered

None.

### PR DESCRIPTION
**Pull request by Nirajn2311**
ISSUES CLOSED: #193

#### Description

Provided a clear and concise description for ‘has’ method in Create a Map Data Structure Challenge and also listed the instructions for the challenge into a bullet list.
Sorry for typo in commit message.

#### Pre-Submission Checklist

- [x] Your pull request targets the `dev` branch.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/challenge-tests`)
- [x] All new and existing tests pass the command `npm test`.
- [x] Use `npm run commit` to generate a conventional commit message.
    Learn more here: <https://conventionalcommits.org/#why-use-conventional-commits>
- [x] The changes were done locally on your machine and NOT GitHub web interface.
    If they were done on the web interface you have ensured that you are creating conventional commit messages.

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #193

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@Nirajn2311, thank you for taking the time to improve this challenge. I especially like the inclusion of the unordered list to make things more readable. I just have one suggestion about the new description for the 'has' method I left as a review. Please give it a read when you can and let me know what you think.

File: challenges/08-coding-interview-prep/data-structures.json [hunk 1]
```diff
@@ -1163,42 +1272,56 @@
...
-        "<code>get</code> accepts a <code>key</code> and returns the stored <code>value</code>",
-        "<code>has</code> returns a <code>boolean</code> for the presence or absence of an item",
-        "<code>values</code> returns an array of all the values in the map",
-        "<code>size</code> returns the number of items in the map",
-        "<code>clear</code> empties the map"
+        "<ul>",
+        "<li><code>add</code> accepts a <code>key, value</code> pair to add to the map.</li>",
+        "<li><code>remove</code> accepts a key and removes the associated <code>key, value</code> pair</li>",
+        "<li><code>get</code> accepts a <code>key</code> and returns the stored <code>value</code></li>",
+        "<li><code>has</code> accepts a <code>key</code> and returns <dfn>true</dfn> or <dfn>false</dfn> based on the presence of a key.</li>",
```
Comment (scissorsneedfoodtoo) [inline challenges/08-coding-interview-prep/data-structures.json]: `"<li><code>has</code> accepts a <code>key</code> and returns <dfn>true</dfn> if an element with the key exists or <dfn>false</dfn> if one doesn't exist.</li>",`

Comment (scissorsneedfoodtoo) [reply challenges/08-coding-interview-prep/data-structures.json]: Or maybe an even shorter version:

``"<li><code>has</code> accepts a <code>key</code> and returns <dfn>true</dfn> if the key exists or <dfn>false</dfn> if it doesn't.</li>",``

Comment (Nirajn2311) [reply challenges/08-coding-interview-prep/data-structures.json]: I think the second one is better.

**Comment by scissorsneedfoodtoo** [comment]
@Nirajn2311, thank you for all of your hard work and for making those changes so quickly! Everything LGTM.

Just to let you know, there was a slight issue due to another recently merged PR that modified the same file, so I went ahead and resolved it.

**Pull request merged**